File: problema07/sol_python/Problema07.py
contador = 2
resultado = 0
numero = float(input("Ingrese hasta que número desea sumar: "))
if numero >= 2:
	while contador <= numero:
		resultado = contador + 2
		contador = contador + 2
else:
	print("Error, el número ingresado es menor a 2."
		+ "\nIntentelo de nuevo.")
print("La suma de los números pares hasta el ", numero, 
	" es: ", resultado)				

File: problema07/sol_java/Problema07/src/problema07/Problema07.java
package problema07;

import java.util.Scanner;
import java.util.Locale;

public class Problema07 {

    public static void main(String[] args) {

        Scanner entrada = new Scanner(System.in);
        entrada.useLocale(Locale.US);

        double numero;
        int contador = 2;
        double resultado = 0;
        System.out.println("Ingrese hasta que número desea sumar: ");
        numero = entrada.nextDouble();

        if (numero >= 2) {
            while (contador <= numero) {
                resultado = contador + 2;
                contador = contador + 2;
            }
        } else {
            System.out.println("Error, el número ingresado es menor a 2."
                    + "\nIntentelo de nuevo.");
        }

        System.out.printf("La suma de los números pares hasta el "
                + "número %.2f es: %.2f\n", numero, resultado);
    }

}
